src/rpc/rawtransaction_util.h

```diff
@@ -19,11 +19,11 @@ class SigningProvider;
  *
  * @param  mtx           The transaction to-be-signed
  * @param  keystore      Temporary keystore containing signing keys
- * @param  coins         Map of unspent outputs - coins in mempool and current chain UTXO set, may be extended by previous txns outputs after call
+ * @param  coins         Map of unspent outputs - coins in mempool and current chain UTXO set
```

Comment:
Could just be "Map of unspent outputs" because "current chain" doesn't seem appropriate since the lock is not held?